src/Movim/i18n/Locale.php

```diff
@@ -179,43 +187,81 @@ public function translate(string $key, $args = false): string
     /**
      * @desc Auto-detects the language from the user browser
      */
-    public function detect($accepted = false)
+    public function detect($accepted = false): ?string
```

Comment:
Small refactor, we can make this parameter strict and call it `?string $language` that fall back to `HTTP_ACCEPT_LANGUAGE` if the value is null, will be way cleaner.